mysql-test/t/audit_log_filter_users_PS-5707.test

```diff
@@ -0,0 +1,63 @@
+# Windows does not have grep
+--source include/not_windows.inc
+
+#
+# PS-5707: Audit log filtering by user is not working
```

Comment:
Do we really need a new testcase for this? The testcase uses the same master parameters as `audit_log_filter_users`, couldn't we include it at the end of that?

Reply:
Moved to audit_log_filter_users